plaso/analyzers/hashers/interface.py

```diff
@@ -12,6 +12,15 @@ class BaseHasher(object):
   NAME = 'base_hasher'
   DESCRIPTION = 'Calculates a digest hash over input data.'
 
+  @classmethod
+  def GetAttributeName(cls):
```

Comment:
move to class constant, the additional indirection here adds no value.

Reply:
Done